package.json

```diff
@@ -1,91 +1,98 @@
-        "karma-source-map-support": "^1.4.0",
-        "karma-typescript": "^5.0.3",
-        "karma-typescript-es6-transform": "^5.0.3",
-        "mocha": "^6.2.1",
-        "mocha-typescript": "1.1.17",
-        "moment": "^2.24.0",
-        "node-cmd": "^3.0.0",
-        "npm-run-all": "^4.1.5",
-        "puppeteer": "^3.3.0",
-        "rimraf": "^3.0.2",
-        "source-map-loader": "^1.0.0",
-        "ts-node": "^8.5.2",
-        "typescript": "^3.7.5",
-        "wait-port": "^0.2.6",
-        "webpack": "^4.43.0",
-        "webpack-cli": "^3.3.12",
-        "yargs": "^15.3.1"
-    }
-}
+{
+    "name": "@autorest/typescript",
+    "version": "6.0.0",
+    "scripts": {
+        "build": "tsc -p .",
+        "build:test:browser": "tsc -p tsconfig.browser-test.json && webpack --config webpack.config.test.js",
+        "check:tree": "ts-node ./test/utils/check-clean-tree.ts",
+        "clean": "rimraf test-browser test/**/node_modules",
+        "start": "node ./dist/src/main.js",
+        "test": "npm-run-all unit-test integration-test",
+        "test:node": "npm run unit-test && npm run start-test-server:v2 & npm run integration-test:alone && npm run stop-test-server",
+        "test:browser": "npm run start-test-server:v2 & npm run integration-test:browser && npm run stop-test-server",
+        "unit-test": "mocha -r ts-node/register './test/unit/**/*spec.ts'",
+        "integration-test": "npm run start-test-server:v2 & npm run generate-and-test && npm run stop-test-server",
+        "integration-test:new": "npm-run-all start-test-server generate-and-test integration-test:alone stop-test-server",
+        "integration-test:browser": "npm run build:test:browser && karma start karma.conf.js",
+        "generate-and-test": "npm-run-all -s build generate-swaggers integration-test:alone integration-test:browser",
+        "integration-test:alone": "mocha -r ts-node/register --timeout 2000 './test/integration/**/*spec.ts'",
+        "start-test-server": "ts-node test/utils/start-server.ts",
+        "start-test-server:v2": "autorest-testserver",
+        "stop-test-server": "autorest-testserver stop",
+        "debug": "node --inspect-brk ./dist/src/main.js",
+        "generate-swaggers": "ts-node ./test/utils/test-swagger-gen.ts",
+        "clone:specs": "git clone https://github.com/Azure/azure-rest-api-specs.git ./.tmp/specs",
+        "smoke-test": "ts-node ./test/utils/smoke-test.ts"
+    },
+    "files": [
+        "src",
+        "dist/**",
+        "README.md",
+        "LICENSE"
+    ],
+    "dependencies": {
+        "@autorest/codemodel": "^4.14.5",
+        "@autorest/extension-base": "^3.2.0",
+        "@azure-tools/codegen": "^2.4.267",
+        "@azure/core-client": "^1.1.2",
+        "@azure/core-http": "^1.2.1-alpha.20201203.2",
+        "@azure/core-https": "^1.0.0-beta.1",
+        "@azure/core-lro": "^1.0.5",
+        "@azure/core-paging": "^1.1.3",
+        "@azure/core-rest-pipeline": "^1.1.0-beta.2",
```

Comment:
We'll need to use ^1.0.3

Reply:
This is intentional. I am expecting some more changes in core-packages. Before releasing the preview version of SDK generator, there will be a core-rest-pipeline GA release (1.0.4) and this will be dependent on that. For now, this version is intentional as this version has a specific bug fix that I did yesterday.